Documentation/building.md

```diff
@@ -22,3 +22,15 @@ You will now have a `flanneld-amd64` binary in the `dist` directory.
     * `make release`
 3. Attach all the files in `dist` to the GitHub release.
 4. Run `make docker-push-all` to push all the images to a registry.
+
+# Obtaining master builds
+
+A new build of flannel is created for every commit to master. They can be obtained from https://quay.io/repository/coreos/flannel-git?tab=tags NOTE: the image name is `coreos/flannel-git`. Releases come from `coreos/flannel`
```

Comment:
Note: The image name is `coreos/flannel-git`, and releases come from `coreos/flannel`.

come from? may be found at?